src/main/java/com/mraof/minestuck/client/gui/RemoteObserverScreen.java

```diff
@@ -8,50 +8,61 @@
 import net.minecraft.client.gui.screen.Screen;
 import net.minecraft.client.gui.widget.TextFieldWidget;
 import net.minecraft.client.gui.widget.button.Button;
-import net.minecraft.client.resources.I18n;
 import net.minecraft.entity.EntityType;
 import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.math.MathHelper;
 import net.minecraft.util.text.StringTextComponent;
 import net.minecraftforge.fml.client.gui.widget.ExtendedButton;
 
 import java.util.Optional;
 
 public class RemoteObserverScreen extends Screen
 {
-	private static final ResourceLocation guiBackground = new ResourceLocation("minestuck", "textures/gui/generic_medium.png");
+	private static final ResourceLocation GUI_BACKGROUND = new ResourceLocation("minestuck", "textures/gui/generic_medium.png");
 	
-	private static final int guiWidth = 150;
-	private static final int guiHeight = 98;
+	private static final int GUI_WIDTH = 150;
+	private static final int GUI_HEIGHT = 98;
 	
 	private final RemoteObserverTileEntity te;
 	private RemoteObserverTileEntity.ActiveType activeType;
+	private int observingRange;
 	
-	public Button typeButton;
+	private Button typeButton;
+	private Button incrementButton;
+	private Button decrementButton;
+	private Button largeIncrementButton;
+	private Button largeDecrementButton;
 	
 	private TextFieldWidget entityTypeTextField;
-	
+	private boolean shouldFinish = true;
```

Comment:
Not a very clean solution, as it's only there to give extra information to the return value of `getEntityType()`. There may be a better way of handling that entity type.